web/src/main/kotlin/no/nav/su/se/bakover/web/routes/søknad/SøknadRoutes.kt

```diff
@@ -74,35 +77,38 @@ internal fun Route.søknadRoutes(
                         call.svar(Feilresponser.ugyldigBody)
                     },
                     ifRight = { søknadsinnholdJson ->
-                        søknadService.nySøknad(
-                            søknadsinnholdJson.toSøknadsinnhold(),
-                            søknadsinnholdJson.forNav.identBruker(call),
-                        ).fold(
-                            { call.svar(it.tilResultat(type)) },
-                            { (saksnummer, søknad) ->
-                                call.audit(
-                                    søknad.søknadInnhold.personopplysninger.fnr,
-                                    AuditLogEvent.Action.CREATE,
-                                    null,
-                                )
-                                call.sikkerlogg("Lagrer søknad ${søknad.id} på sak ${søknad.sakId}")
-                                SuMetrics.søknadMottatt(
-                                    if (søknad.søknadInnhold.forNav is ForNav.Papirsøknad)
-                                        SuMetrics.Søknadstype.PAPIR
-                                    else
-                                        SuMetrics.Søknadstype.DIGITAL,
-                                )
-                                call.svar(
-                                    Resultat.json(
-                                        Created,
-                                        serialize(
-                                            OpprettetSøknadJson(
-                                                saksnummer = saksnummer.nummer,
-                                                søknad = søknad.toJson(),
-                                            ),
-                                        ),
-                                    ),
-                                )
+                        søknadsinnholdJson.toSøknadsinnhold().fold(
+                            { call.svar(it.tilResultat()) },
```

Comment:
Dersom du ønsker mer compiler støtte for å se at man utfører call.svar i alle conditions, kan man lagre resultat til en felles variabel eller gjøre en let og kun ha et `call.svar`